src/NodaTime.Test/TimeZones/BclDateTimeZoneTest.cs

```diff
@@ -267,6 +268,50 @@ public void FakeDaylightSavingTime()
             Assert.AreEqual(expectedSummer, summerInterval);
         }
 
+        // See https://github.com/nodatime/nodatime/issues/1524 for the background
+        // It would be nice to test earlier dates (e.g. 1967 and 1998) where transitions
+        // actually occurred on the first of the "next month", but the Windows database has very different
+        // data from TZDB in those cases.
+        [Test]
+        public void TransitionAtMidnight()
+        {
+            var bclZone = GetBclZoneOrIgnore("E. South America Standard Time");
+            var nodaTzdbZone = DateTimeZoneProviders.Tzdb["America/Sao_Paulo"];
+            var nodaBclZone = BclDateTimeZone.FromTimeZoneInfo(bclZone);
+
+            var expectedFallBack = Instant.FromUtc(2012, 2, 26, 2, 0, 0);
+            var expectedSpringForward = Instant.FromUtc(2012, 10, 21, 3, 0, 0);
+
+            // We assume that the fall-back happens before the spring-forward in the dates
```

Comment:
Unfinished comment? (Also, do we have to assume anything? This test is for a concrete situation.)

Reply:
Thanks, will address.